### PR TITLE
Adds test for strip_tags string filter

### PR DESCRIPTION
**Pull request by hhatfield**
This adds coverage for the constructor and destructor of the strip_tags filter. Also tests the sample code from http://php.net/manual/en/filters.string.php

UPHPU TestFest

### CODE REVIEW

File: ext/standard/tests/filters/strip_tags.phpt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+--TEST--
+strip_tags filter - ctor and dtor
+--CREDIT--
+H Hatfield hatfieldje@gmail.com UPHPU Test Fest
+--FILE--
+<?php
+# vim600:syn=php:
```
Comment (carusogabriel) [inline ext/standard/tests/filters/strip_tags.phpt:7]:
```suggestion
```

**Review by carusogabriel — CHANGES_REQUESTED** [review]

File: ext/standard/tests/filters/strip_tags.phpt (hunk 1 above)
Comment (petk) [reply ext/standard/tests/filters/strip_tags.phpt:7]: This one is the only change needed?

Comment (hhatfield) [reply ext/standard/tests/filters/strip_tags.phpt:7]: This is copy/paste from whichever test I used as a template. It seems to show up in quite a few files. I have no strong ties to this line, but would like to understand why it should be removed.

Comment (petk) [reply ext/standard/tests/filters/strip_tags.phpt:7]: @hhatfield These are so called Vim modes. They are now gone from the php-src in favour of the EditorConfig and manual user side configurations. See this change:
https://github.com/php/php-src/commit/92ac598aabd336593a47ed3959f1031674b763e6

and here https://externals.io/message/101206

Yes, they were present once in some of those files. Some are still there but only on very rare places (for eample *.w32 files to enable syntax highlighting and for some documentation purposes). Long term: they should all be replaced with different approaches. For example, EditorConfig once more options will be available.

**Comment by petk** [comment]
@carusogabriel if all other things here are good, I'll merge this one also then... Maybe also `--EXPECTF--` to `--EXPECT--` change will be done, other things seem good to add.

**Comment by carusogabriel** [comment]
@petk Go ahead then.

**Comment by petk** [comment]
@carusogabriel the [string.strip_tags](http://php.net/manual/en/filters.string.php) has been deprecated in PHP 7.3... 🙄

**Comment by carusogabriel** [comment]
If so, @hhatfield thanks for the PR, but it's not valid anymore :confused:

**Pull request closed without merging**